src/SectorView.h

```diff
@@ -95,11 +94,12 @@ class SectorView : public UIView, public DeleteEmitter {
 		Axis *mapViewYaw;
 		Axis *mapViewPitch;
 		Axis *mapViewZoom;
-
+		virtual void RegisterBindings();
```

Comment:
Tiny minor tweak, but please prefer the use of `override` instead of declaring the function as virtual... it makes the intent much clearer:
```suggestion
		void RegisterBindings() override;
```

Reply:
Yes, that's much better, thanks! Somehow I did not pay attention to this keyword, although I used it in Java.